server/test/unit/server/async/test_scheduler.py

```diff
@@ -506,7 +506,7 @@ def test_debug_logging(self, mock__logger, mock_worker, mock_delete_worker):
         scheduler.CeleryProcessTimeoutMonitor().check_celery_processes()
         mock__logger.debug.assert_has_calls([
             mock.call('Checking if pulp_workers, pulp_celerybeat, or '
-                      'pulp_resource_manager processes are missing for more than 300 seconds'),
+                      'pulp_resource_manager processes are missing for more than 60 seconds'),
```

Comment:
Can this use the variable value instead so that we can stay DRY. Yes I know I wrote this code originally. :-)